src/App.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import logo from './logo.svg';
+import './App.css';
+import Login from './components/Login/login.js';
+import CanvasJSReact from './canvas/canvasjs.react.js';
+//var CanvasJSReact = require('./canvasjs.react');
+var CanvasJS = CanvasJSReact.CanvasJS;
+var CanvasJSChart = CanvasJSReact.CanvasJSChart;
+
+
+
+function App() {
+  return (
+
+    <Login/>
```

Comment:
what changes did you make here?

Reply:
in Dashboard.js I redid modal and made no changes to App.js

Reply:
this App.js was the old one merged from master i think.. getting rid of it